darknet_ros/src/YoloObjectDetector.cpp

```diff
@@ -263,7 +263,7 @@ bool YoloObjectDetector::publishDetectionImage(const cv::Mat& detectionImage)
     return false;
   cv_bridge::CvImage cvImage;
   cvImage.header = headerBuff_[buffRdInd_];
-  cvImage.encoding = sensor_msgs::image_encodings::BGR8;
+  cvImage.encoding = sensor_msgs::image_encodings::RGB8;
```

Comment:
is this a change of img encoding?

Reply:
For the visualization yes

Reply:
Apparently the visualization was not correct anymore after the changes. We can also test again on the robot to double check.

Reply:
It might be related to how we create `mat_`

Reply:
Ok changes in https://github.com/ntnu-arl/darknet_ros/pull/4/commits/74e43248d63e9222e3b3cc9563c19f87b5850be1 fix the requirement for the change in color coding.